gci/componentmodel.py

```diff
@@ -539,10 +554,6 @@ def __post_init__(self):
             self.access = AccessDict(access)
 
 
-# backwards-compatibility
-ComponentSource = Source
```

Comment:
There are still references in delivery-service (did **not** check other repos)

Reply:
good catch - let us rename those, too (this should be a trivial search-and-replace task. I did check our "internal" repositories, though.